conf/modules.config

```diff
@@ -45,22 +37,19 @@ process {
         publishDir = [
             [
                 path: { "${params.outdir}/trimgalore/fastqc" },
-                mode: 'copy',
+                mode: params.publish_dir_mode,
```

Comment:
Shouldn't this also be moved to lines 16-17? (In several places.)

Reply:
it is actually there (line 15) maybe we can get rid on all the other places then.

Reply:
I think so.